CMakeLists.txt

```diff
@@ -49,6 +49,22 @@ if (${BUILD_VIEWER})
     COMMAND npm run build
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/view
   )
+  install(DIRECTORY view/build DESTINATION /var/lib/diffkemp/view
+    # Setting 755 for files.
+    FILE_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
+                     GROUP_READ GROUP_EXECUTE
+                     WORLD_READ WORLD_EXECUTE
+    # Setting 777 for directories to make it possible to create
+    # `src` and `diffs` folder by user (`diffkemp view`).
+    DIRECTORY_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
+                     GROUP_READ GROUP_WRITE GROUP_EXECUTE
+                     WORLD_READ WORLD_WRITE WORLD_EXECUTE
+    # Making it possible to remove `diffkemp-out.yaml`
+    # by a user (`diffkemp view`) if it existed during installation.
```

Comment:
The file should not exist during installation. If it does, let's not install it.

Reply:
There was a problem with creating `diffkemp-out.yaml` because of the set directory permissions, I decided to create this file (empty) when installing, and `diffkemp view` will write to it and after it exits it will make the file again empty.

Reply:
Ah, I see, that's the one file created by `diffkemp view `. Ok, let's stick with an empty file for now but I'd like to handle that properly later.